images/linux/Ubuntu2204-Readme.md

```diff
@@ -78,51 +77,51 @@ to accomplish this.
 - Docker-Buildx 0.10.4
 - Docker-Moby Client 20.10.23+azure-2
 - Docker-Moby Server 20.10.23+azure-3
-- Fastlane 2.212.1
+- Fastlane 2.212.2
 - Git 2.40.0
 - Git LFS 3.3.0
 - Git-ftp 1.6.0
 - Haveged 1.9.14
 - Heroku 7.69.1
 - jq 1.6
 - Kind 0.18.0
-- Kubectl 1.26.3
+- Kubectl 1.27.1
 - Kustomize 5.0.1
 - Leiningen 2.10.0
 - MediaInfo 21.09
 - Mercurial 6.1.1
 - Minikube 1.30.1
-- n 9.0.1
+- n 9.1.0
 - Newman 5.3.2
 - nvm 0.39.3
 - OpenSSL 3.0.2-0ubuntu1.8
-- Packer 1.8.6
+- Packer Packer 1.8.6
```

Comment:
Look like we need fix Get-PackerVersion function in https://github.com/actions/runner-images/blob/5657dec7af3b55531d12f753cbd04370357941de/images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1

Reply:
is that related https://github.com/actions/runner-images/pull/7431/files ?